src/services/redisSessionStorage.ts

```diff
@@ -107,13 +119,25 @@ export default class RedisSessionStorage implements ISessionStorage {
     sessionToken: SessionToken,
     walletToken: WalletToken
   ): Promise<Either<Error, boolean>> {
-    const deleteSessionToken = new Promise<Either<Error, boolean>>(resolve => {
-      // Remove the specified key. A key is ignored if it does not exist.
-      // @see https://redis.io/commands/del
-      this.redisClient.del(
-        `${sessionKeyPrefix}${sessionToken}`,
-        (err, response) => resolve(this.integerReply(err, response))
-      );
+    const keys = await new Promise<Either<Error, ReadonlyArray<string>>>(
+      resolve => {
+        this.redisClient.keys(
+          `*${sessionKeyPrefix}${sessionToken}`,
+          (err, response) => resolve(this.arrayStringReplay(err, response))
+        );
+      }
+    );
+    if (isLeft(keys)) {
+      return this.integerReply(keys.value, undefined);
+    }
+    const deleteSessionTokens = keys.value.map(key => {
```

Comment:
you don't need a `map`, you [can delete multiple keys at once](https://redis.io/commands/del)

Reply:
fixed on 2354a0b